app/controllers/api/v1/subject_sets_controller.rb

```diff
@@ -105,7 +105,9 @@ def add_relation(resource, relation, value)
         [ resource.id, subject_id, rand ]
       end
 
-      SetMemberSubject.import IMPORT_COLUMNS, new_sms_values, validate: false
+      result = SetMemberSubject.import IMPORT_COLUMNS, new_sms_values, validate: false
+      SubjectPriorityWorker.perform_async(result.ids)
```

Comment:
Do we have an idea about how big this array could be? https://github.com/mperham/sidekiq/wiki/Best-Practices#1-make-your-job-parameters-small-and-simple

How can we minimize the memory pressure on redis?

Reply:
Had a bunch written here about whether a one job with many arguments was better than many jobs with one argument when I realized that I can just send the subject set ID and pull the SMSes out again in the worker. 🎉